skelebot/objects/arg.py

```diff
@@ -0,0 +1,23 @@
+from .skeleYaml import SkeleYaml
+
+class Arg(SkeleYaml):
+    name = None
+    choices = None
+    help = None
+
+    def __init__(self, name, choices=[], help=None):
```

Comment:
you shouldn't have empty lists as function parameters. Python doesn't like that: http://effbot.org/zone/default-values.htm